actioncable/app/javascript/actioncable/index.js

```diff
@@ -0,0 +1,47 @@
+import { Consumer } from "./consumer"
+
+let defaultExport = {}
+defaultExport.ActionCable = {
+  WebSocket: window.WebSocket,
+  logger: window.console,
+
+  createConsumer(url) {
+    if (url == null) { let left
+    url = (left = this.getConfig("url")) != null ? left : this.INTERNAL.default_mount_path }
```

Comment:
Is `INTERNAL` defined anywhere? Will need some kind of setup to replace: https://github.com/rails/rails/blob/45903be1344992b1f0156cd88c351067bdc93726/actioncable/app/assets/javascripts/action_cable.coffee.erb#L5-L6